packages/docs/src/pages/tutorial/store/basic/index.mdx

```diff
@@ -3,20 +3,21 @@ title: Storing State
 layout: tutorial
 ---
 
-Applications need to store state to be useful. (Otherwise, they are just static pages.) 
+Qwik tracks application state for two reasons:
 
-Qwik needs to track the application state for two reasons:
-1. Application state needs to be serialized on application pause (and deserialize on application resume.)
-2. Qwik needs to create subscriptions on stores so that it can know which components need to be re-rendered (if Qwik would not track subscription information then it would have to re-render the whole application which would defeat the lazy-loading.)
+1. To serialize data when the application pauses, and deserialize as the application resumes.
```

Comment:
I would modify it to  "pauses on the server" and "resumes on the client"

Reply:
Thanks! That keeps it much more explicit